index.html

```diff
@@ -15,6 +15,9 @@
 		<header>
 			<h1><a href="?repo=[repo]" title="Click for permalink">GitHub Issue Closing Times</a></h1>
 			<form mv-action="set(repo, repoInput), set(labels, labelFilter)">
+				<p class="mv-message mv-warning mv-inline" mv-if="!(len(repoInput) >= 3 and contains(repoInput, '/'))">
+					The entered repository name is not valid. Please enter it in the format <code>organisation/repository</code>
```

Comment:
```suggestion
					The entered repository name is not valid. Please enter it in the format <code>owner/repository</code>
```

"Owner" encompasses both users and orgs